.env.testing

```diff
@@ -0,0 +1,12 @@
+APP_NAME="AIC Mobile Admin"
+APP_ENV=testing
+APP_KEY=base64:mtTLixAtmR5vwu2J/GB1+NHqqopjFNFE1pJhZ47dyLg=
+APP_DEBUG=true
+APP_URL='http://localhost'
+
+DB_CONNECTION='mysql'
+DB_HOST='127.0.0.1'
+DB_PORT=3306
+DB_DATABASE='mobile-admin'
```

Comment:
This should be named something that explicitly notes this as a test database. Perhaps `mobile-admin-test`?

Also, the .env.example file suggests `mobile-cms`. In `utils` [we landed on `mobile-admin`](https://github.com/art-institute-of-chicago/utils/commit/e7e6551859599a40580f8deb66cf3c47ad4efcc5) as the app/database name, so that reference should be updated as well.

Reply:
Corrected